src/systems/filecoin_mining/storage_mining/storage_miner_actor_state.go

```diff
@@ -40,6 +40,10 @@ func (st *StorageMinerActorState_I) _shouldChallenge(currEpoch block.ChainEpoch)
 	return st.ChallengeStatus().ShouldChallenge(currEpoch)
 }
 
+func (st *StorageMinerActorState_I) _verifySeal(rt Runtime, onChainInfo sector.OnChainSealVerifyInfo) bool {
```

Comment:
It's a monster of a function but I think this may be redundant with `ProveCommitSector` (could be factored out of there so PCS is a shorter method) though I can't tell for sure. @zixuanzh ?